docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -136,33 +141,34 @@ and execute the ready to run file.
 
 The following section types are defined and described later in this document:
 
-| ReadyToRunSectionType                          | Value | Scope (component assembly / entire image)
-|:-----------------------------------------------|------:|:-----------
-| READYTORUN_SECTION_COMPILER_IDENTIFIER         |   100 | Image
-| READYTORUN_SECTION_IMPORT_SECTIONS             |   101 | Image
-| READYTORUN_SECTION_RUNTIME_FUNCTIONS           |   102 | Image
-| READYTORUN_SECTION_METHODDEF_ENTRYPOINTS       |   103 | Assembly
-| READYTORUN_SECTION_EXCEPTION_INFO              |   104 | Assembly
-| READYTORUN_SECTION_DEBUG_INFO                  |   105 | Assembly
-| READYTORUN_SECTION_DELAYLOAD_METHODCALL_THUNKS |   106 | Assembly
-| ~~READYTORUN_SECTION_AVAILABLE_TYPES~~         |   107 | (obsolete - used by an older format)
-| READYTORUN_SECTION_AVAILABLE_TYPES             |   108 | Assembly
-| READYTORUN_SECTION_INSTANCE_METHOD_ENTRYPOINTS |   109 | Image
-| READYTORUN_SECTION_INLINING_INFO               |   110 | Assembly (added in V2.1)
-| READYTORUN_SECTION_PROFILEDATA_INFO            |   111 | Image (added in V2.2)
-| READYTORUN_SECTION_MANIFEST_METADATA           |   112 | Image (added in V2.3)
-| READYTORUN_SECTION_ATTRIBUTEPRESENCE           |   113 | Assembly (added in V3.1)
-| READYTORUN_SECTION_INLINING_INFO2              |   114 | Image (added in V4.1)
-| READYTORUN_SECTION_ASSEMBLIES                  |   115 | Image (added in V4.1)
-
-## READYTORUN_SECTION_COMPILER_IDENTIFIER
+| ReadyToRunSectionType     | Value | Scope (component assembly / entire image)
+|:--------------------------|------:|:-----------
+| CompilerIdentifier        |   100 | Image
+| ImportSections            |   101 | Image
+| RuntimeFunctions          |   102 | Image
+| MethodDefEntryPoints      |   103 | Assembly
+| ExceptionInfo             |   104 | Assembly
+| DebugInfo                 |   105 | Assembly
+| DelayLoadMethodCallThunks |   106 | Assembly
+| ~~AvailableTypes~~        |   107 | (obsolete - used by an older format)
+| AvailableTypes            |   108 | Assembly
+| InstanceMethodEntryPoints |   109 | Image
+| InliningInfo              |   110 | Assembly (added in V2.1)
+| ProfileDataInfo           |   111 | Image (added in V2.2)
+| ManifestMetadata          |   112 | Image (added in V2.3)
+| AttributePresence         |   113 | Assembly (added in V3.1)
+| InliningInfo2             |   114 | Image (added in V4.1)
+| ComponentAssemblies       |   115 | Image (added in V4.1)
+| OwnerCompositeExecutable  |   116 | Image (added in V4.1)
+
+## CompilerIdentifier
 
 This section contains zero terminated ASCII string that identifies the compiler used to produce the
 image.
 
 **Example**: `CoreCLR 4.6.22727.0 PROJECTK`
 
-## READYTORUN_SECTION_IMPORT_SECTIONS
+## ImportSections
```

Comment:
Nit: I may be nice to include the fully qualified names in these chapters, e.g. `ReadyToRunSectionType.ImportSections`

Reply:
Good idea, will fix.

Reply:
Fixed in 2nd commit.